Commands/Site/AddRoleDefinition.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System.Management.Automation;
+using Microsoft.SharePoint.Client;
+using SharePointPnP.PowerShell.CmdletHelpAttributes;
+using System.Collections.Generic;
+using SharePointPnP.PowerShell.Commands.Base.PipeBinds;
+
+namespace SharePointPnP.PowerShell.Commands.Site
+{
+    [Cmdlet(VerbsCommon.Add, "PnPRoleDefinition")]
+    [CmdletHelp("Adds a Role Defintion (Permission Level) to the site collection in the current context",
+        DetailedDescription = "This command allows adding a custom Role Defintion (Permission Level) to the site collection in the current context. It does not replace or remove existing Role Definitions.",
+        Category = CmdletHelpCategory.Sites)]
+    [CmdletExample(
+        Code = @"PS:> Add-PnPRoleDefinition -RoleName ""CustomPerm""",
+        Remarks = @"Creates additional permission level with no permission flags enabled.", SortOrder = 1)]
+    [CmdletExample(
+        Code = @"PS:> Add-PnPRoleDefinition -RoleName ""NoDelete"" -Clone ""Contribute"" -Exclude ""DeleteListItems""",
+        Remarks = @"Creates additional permission level by cloning ""Contribute"" and removes flags ""DeleteListItems""", SortOrder = 2)]
+    [CmdletExample(
+        Code = @"PS:> Add-PnPRoleDefinition -RoleName ""AddOnly"" -Clone ""Contribute"" -Exclude ""DeleteListItems,EditListItems""",
+        Remarks = @"Creates additional permission level by cloning ""Contribute"" and removes flags ""DeleteListItems,EditListItems""", SortOrder = 3)]
+
+    public class AddSiteCollectionAdmin : PnPCmdlet
```

Comment:
You name the cmdlet Add-PnPRoleDefinition, however the class you created is called AddSiteCollectionAdmin.

Reply:
If you fix this and update this PR I gladly merge it.